gallia-prehistoire.csl

```diff
@@ -239,6 +242,16 @@
         </choose>
         <text macro="issue" prefix=", "/>
       </if>
+      <else-if type="chapter entry-dictionary entry-encyclopedia" match="any">
+       <group delimiter=", ">
+       <text macro="publisher-place" font-style="italic"/>
+       </group>
+      </else-if>
+      <else-if type="manuscript thesis report" match="any">
+       <group delimiter=", ">
+       <text macro="publisher-place" font-weight="normal"/>
```

Comment:
@tomax31, here, and at line 247 above, you call a non-existing "publisher-place" macro, which is why our automatic testing is giving an error. (if you just want to print the "publisher-place" variable, you can use `<text variable="publisher-place" ... />` instead of `<text macro="publisher-place" ... />`)